Add Note to Address Worker Memory Leak (#745)
astronomer/docs: astro/deployment-metrics.md

**Worker CPU** records the peak CPU usage, while **Worker Memory** records the peak memory usage by worker nodes over hour-long intervals. The bolded values above each graph show the maximum CPU/ memory usage by a single worker at any point in time over the last 24 hours.

> > **Note:** A known issue with Celery may cause worker memory allocation to increase without a corresponding increase in task count or DAG memory use. To correct this issue, run a no operation deploy to restart the Celery worker and reset memory requests. See [Deploy Code to Astro](deploy-code.md). This issue should not affect your tasks or cause OutOfMemory (OOM) errors. If you experience complications, contact Astronomer Support. 
> 

Reviewer: ```suggestion
:::info

A known issue with Celery might cause worker memory allocation to increase without a corresponding increase in task count or DAG memory use. To correct this issue, [redeploy your code](deploy-code.md) with no changes to restart the Celery worker and reset memory requests. This issue should not affect your tasks or cause OutOfMemory (OOM) errors. If you experience complications, contact [Astronomer Support](https://support.astronomer.io/). 

:::
```

Reviewer: Few things here:

- `>Note` style notes are a bit of legacy formatting from way back when, we use `info` notes now as a 1:1 replacement for them.
- I removed "no operation" here because it's not immediately clear what that means in the context of Astronomer deploys. Tried instead to describe the exact action that someone would complete. 
- Tried to make the link to Deploy Code sound a little more conversational and flow with the text. If this sort of unacknowledged link has accessibility considerations, I'd instead write "To correct this issue, redeploy your same code as described in [Deploy Code](deploy-code.md) to restart the Celery worker and reset memory requests."

Author: Thanks @jwitz for the quick review.

I updated the Note to the Info style and modified the text to fit with your recommendations.
